expect: add page (#3441)
tldr-pages/tldr: pages/linux/expect.md

# expect

> > Programmed dialogue with interactive programs.

Reviewer: Hrm, this is an awkward one because this is what the man page says, but it's a bit on the confusing side. Would something like this still accurately reflect what the command does, @einverne?
```suggestion
> Script executor that interacts with other programs that require user input.
```

> > More intermation <https://linux.die.net/man/1/expect>.

Reviewer: Missing colon & typo :-)
```suggestion
> More information: <https://linux.die.net/man/1/expect>.
```

> - Execute expect scripts from file:

Reviewer: Perhaps this will read better?
```suggestion
- Execute an expect script from a file:
```

Also, could `{{file}}` on the following line be `{{path/to/file}}` please?

`expect {{file}}`

> - Execute expect commands from text:

Reviewer: Perhaps this will read better?
```suggestion
- Execute a specified expect script:
```

Also, we might want to wrap `{{commands}}` in quotes.

> `expect {{/path/to/file}}`

Reviewer 2: Remove preceding `/` from /path/to/file

`expect -c {{commands}}`

> - Enter expect interactive mode (use "exit" or "EOF" to exit):

Reviewer: Perhaps this will read better?
```suggestion
- Enter an interactive REPL (use `exit` or Ctrl + D to exit):
```
